package.json

```diff
@@ -242,7 +242,7 @@
     "@expo/config-plugins/glob": "^10.3.10",
     "@solana/web3.js/rpc-websockets": "^8.0.1",
     "@metamask/nonce-tracker@npm:^5.0.0": "patch:@metamask/nonce-tracker@npm%3A5.0.0#~/.yarn/patches/@metamask-nonce-tracker-npm-5.0.0-d81478218e.patch",
-    "@metamask/network-controller@npm:^22.0.0": "patch:@metamask/network-controller@npm%3A22.1.0#~/.yarn/patches/@metamask-network-controller-npm-22.1.0-621c281f70.patch",
+    "@metamask/network-controller@npm:^22.0.0": "patch:@metamask/network-controller@npm%3A22.1.1#~/.yarn/patches/@metamask-network-controller-npm-22.1.0-621c281f70.patch",
```

Comment:
The patch is still based on 22.1.0 🤔

Reply:
Ops! updated in 58c5bac5538622c497ab0fb0c08d07c20d143b2a